### ISSUE TITLE
Landing Page: <Include font in the styles page>

### ISSUE THREAD

**Issue by stuartkirkham** [TheOdinProject/curriculum#28839]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

It would be helpful to place the font being used in the project in the picture image of the colors and font styles! I missed that part in my first read through of the instructions but i feel it would be helpful to be in that image as that includes the other font styling and color instructions.

### Path

Foundations

### Lesson Url

https://www.theodinproject.com/lessons/foundations-landing-page

**Comment by CouchofTomato** [comment]
@TheOdinProject/foundations Can someone take a look please.

**Comment by MaoShizhong** [comment]
It's a reasonable suggestion given that it's a fairly common question in community. You'd also expect an image labelled "colors and fonts" to contain font information and not just font size/weight information.

## Acceptance criteria

This is a 2 PR process, following our [guide for adding images to the curriculum](https://github.com/TheOdinProject/curriculum/wiki/Adding-Images-to-the-Curriculum).

- [ ] The "colors and fonts" image is replaced with an image that also includes the font used (Roboto).
- [ ] The assignment substep stating Roboto is the font used is removed as a result.

Please comment below if you'd like to be assigned to this issue. Please do not open a PR unless you have been assigned by a maintainer.

**Comment by forge34** [comment]
i would like to work on this issue if possible

**Comment by MaoShizhong** [comment]
All yours then
